### PR TITLE
Build: Add Node 6.x to Travis

### PR DESCRIPTION
**Pull request by dannyfritz**
Finishing what #57 started.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,5 +3,8 @@ sudo: false
 node_js:
     - "0.12"
     - 4
+    - 6
+before_script:
+    - npm install typescript@^1.7.3
```
Comment (nzakas) [inline .travis.yml]: Any way we can pull the version from package.json so we don't have to worry about keeping the versions in sync?

Comment (dannyfritz) [inline .travis.yml]: We could add this as a `devDependency` and run it instead. https://www.npmjs.com/package/npm-install-peers

**Comment by nzakas** [comment]
Also, this should be "Build:"

**Comment by dannyfritz** [comment]
Oh geez, 0.12 can't run `npm-install-peers`:

```sh
const chalk = require('chalk')
^^^^^
SyntaxError: Use of const in strict mode.
```

**Comment by dannyfritz** [comment]
Working with `npm-install-peers` to resolve this: https://github.com/spatie/npm-install-peers/issues/2

**Comment by JamesHenry** [comment]
@nzakas I am wondering if node 0.12 should even be a consideration at this point... It's maintenance support will be discontinued at the end of the year, and node 4.x has long since been the LTS.

It would really surprise me if there are many users of TypeScript and node 0.12 as a combination out there.

What do you think?

**Comment by dannyfritz** [comment]
Got a PR in for `npm-install-peers` and this should be good to go now.

https://github.com/spatie/npm-install-peers/pull/3

**Pull request merged**